earthpy/tests/test_plot.py

```diff
@@ -5,7 +5,6 @@
 import earthpy.spatial as es
 # For builds on travis to avoid plot display errors
 import matplotlib as mpl
```

Comment:
Looks like `mpl` is not called anymore - can we delete lines 6 and 7?

Reply:
done!